vehicle/OVMS.V3/components/vehicle_smarteq/src/vehicle_smarteq.cpp

```diff
@@ -506,8 +522,8 @@ void OvmsVehicleSmartEQ::HandlePollState() {
 
 void OvmsVehicleSmartEQ::CalculateEfficiency() {
   float consumption = 0;
-  if (StdMetrics.ms_v_pos_speed->AsFloat() >= 5)
-    consumption = ABS(StdMetrics.ms_v_bat_power->AsFloat(0, Watts)) / StdMetrics.ms_v_pos_speed->AsFloat();
+  if (StdMetrics.ms_v_pos_gpsspeed->AsFloat() >= 5)
+    consumption = ABS(mt_bms_BattPower_power->AsFloat(0, kW)) / StdMetrics.ms_v_pos_trip->AsFloat();
   StdMetrics.ms_v_bat_consumption->SetValue((StdMetrics.ms_v_bat_consumption->AsFloat() * 4 + consumption) / 5);
```

Comment:
Was machen die leute ohne Modem und GPS? Also WIFI only würde das nicht Funktionieren. Zudem könnte ich die Funktion auch löschen da es ein Overwrite aus der vehicle.cpp ist.
Zudem macht pos_trip da überhaupt keinen sinn. Da es die Berechnete Effizienz von speed und Power ist.

Es kann sein das der ms_v_bat_power falsch ist. Muss ich mal gucken.

Reply:
da bin ich eh eher am testen was das bringen soll, in der App wird's eh nicht angezeigt (kein Log Eintrag). Daher auch eher nice to have als must have.

Reply:
Glaube der wert ist für die Dashboard anzeige in der Web Oberfläche